### ISSUE TITLE
attachment details disappear when preview and back

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1713]
Same device as the linked issue.

Reproduce:

1.    go to Contacts app on Android, "manage", "export all". It produces a vcf file, like contacts.vcf
2.   use FlowCrypt for Android to send it to yourself encrypted
3.   open on iOS and notice the attachment on the bottom
4.   the contents of the file render as text in the "view attachment" screen
5. tap the back icon on top left
6. message shows but it's now missing attachment details

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/issues/1707#issuecomment-1161519929_

**Comment by ioanmo226** [comment]
I can't reproduce this issue either.
@sosnovsky Could you check if it happens in your side?

**Comment by sosnovsky** [comment]
> I can't reproduce this issue either.
> @sosnovsky Could you check if it happens in your side?

Yes, I'll try to reproduce it on my device. Maybe it depends on file name (so only long names disappear)?

**Comment by ioanmo226** [comment]
Yeah, I agree.
But I wonder why it only happens after user goes back to screen.

Need more details from @tomholub though.
